sandbox/functions-recipes/includes/durable-authoring-activities.md

```diff
@@ -0,0 +1,26 @@
+## Authoring Activity Functions
+Activities are composed together in orchestration functions to create a stateful workflow. Activities are where most of the processing happens for a given orchestration. `ActivityTriggerAttribute` is used to annotate functions as orchestration activities.
```

Comment:
It's subtle, but I typically use the term "orchestrat**or** function" rather than "orchestrat**ion** function". In most other places you seem to be using "orchestrator" to describe the function type correctly.